recipe/meta.yaml

```diff
@@ -10,34 +10,40 @@ source:
   sha256: 22f854a5fcab42b5ddba8030a0e4be51ca89af0267961c8d6cfa86395586c407
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
 
 requirements:
   host:
-    - python >=2.7
+    - python
     - pip
+    - wheel
+    - setuptools
   run:
-    - python >=2.7
+    - python
     - python-dateutil >=2.6.0
     - ruamel.yaml >=0.17
```

Comment:
Does it actually make use of ruamel.yaml?
This looks like a fork: https://github.com/crdoconnor/strictyaml/tree/1.7.3/strictyaml/ruamel

Reply:
It does yes. I initially removed and and then searched a bit deeper and saw that it imports the optimized ruamel modules, see https://github.com/crdoconnor/strictyaml/blob/1.7.3/strictyaml/ruamel/main.py#L57.

Reply:
good catch!

Reply:
No, it doesn't depend on ruamel.yaml because it includes the [package](https://github.com/crdoconnor/strictyaml/tree/1.7.3/strictyaml/ruamel) `ruamel` as a source code of v0.16.13 https://github.com/crdoconnor/strictyaml/blob/1.7.3/strictyaml/ruamel/__init__.py#L11

Reply:
@skupr-anaconda I depends on it. Like I said in my previous reply, it will use the C accelerated ruamel modules if they are present. See https://github.com/crdoconnor/strictyaml/blob/1.7.3/strictyaml/ruamel/main.py#L57.

Reply:
0.17 comes from conda-forge. But I think it should be 0.16.13 as per https://github.com/crdoconnor/strictyaml/blob/1.7.3/strictyaml/ruamel/__init__.py#L11.

Reply:
Though, I have to say that it changes nothing.